calico_node/startup/startup.go

```diff
@@ -830,12 +830,12 @@ func ensureDefaultConfig(ctx context.Context, cfg *apiconfig.CalicoAPIConfig, c
 		}
 	} else {
 		updateNeeded := false
-		if felixConf.Spec.ReportingIntervalSecs == nil {
-			interval := 0
-			felixConf.Spec.ReportingIntervalSecs = &interval
+		if felixConf.Spec.ReportingInterval == nil {
+			interval := metav1.Duration{Duration: 0}
+			felixConf.Spec.ReportingInterval = &interval
```

Comment:
you can probably one line this into `felixConf.Spec.ReportingInterval = &metav1.Duration{Duration: 0}`